teamwork/static/css/main.css

```diff
@@ -1,6 +1,43 @@
+/* Global Styles */
+
 html {
   box-sizing: border-box;
 }
 *, *:before, *:after {
   box-sizing: inherit;
 }
+
+body {
+    margin-top: 0px;
+    background-color: #fff;
+}
+
+@media(min-width:768px) {
+    body {
+        margin-top: 0px;
+    }
+}
+
+#page-wrapper {

```

Comment:
Doesn't the bootstrap `container` class do already the same thing as the `page-wrapper` class?